Python/jit.c

```diff
@@ -102,8 +107,11 @@ mark_executable(unsigned char *memory, size_t size)
     int old;
     int failed = !VirtualProtect(memory, size, PAGE_EXECUTE_READ, &old);
 #else
+    int failed = 0;
     __builtin___clear_cache((char *)memory, (char *)memory + size);
-    int failed = mprotect(memory, size, PROT_EXEC | PROT_READ);
+# ifndef MAP_JIT
+    failed = mprotect(memory, size, PROT_EXEC | PROT_READ);
+# endif
```

Comment:
Since this is only one line, the nesting is clear, I think.

```suggestion
#ifndef MAP_JIT
    failed = mprotect(memory, size, PROT_EXEC | PROT_READ);
#endif
```

Reply:
Done